publish.css

```diff
@@ -1,23 +1,112 @@
 /* This Theme is called "Reverie" and Obsidian Theme created by Santi Younger */
-
-Last Update:
-2021-03-11 12:06
-
 @import url('https://fonts.googleapis.com/css2?family=Modern+Antiqua&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
 
-:root {
-  --font-monospace: 'Source Code Pro', monospace;
-
+body {
+  --font-monospace: 'Source Code Pro';
   --wws-font-modern: "Modern Antiqua";
-
   --wws-font-raleway: "Raleway";
```

Comment:
I left this one here, but currently its not used in anything.

Reply:
Good catch, I implemented it 😄